app/assets/stylesheets/pages/_home.scss

```diff
@@ -59,48 +59,59 @@ img {
   border: 0;
 }
 
-/* Navbar search form */
-.navbar-wagon-search {
-  flex: 0 1 300px;
-  display: flex;
-  justify-content: flex-end;
-  padding: 0 10px;
-  position: relative;
-  margin-top: 50px;
-  text-align: center;
-  display: inline-block;
-  left: 525px;
-}
-.navbar-wagon-search-2 {
-  flex: 0 1 300px;
-  display: flex;
-  justify-content: flex-end;
-  padding: 0 10px;
-  position: relative;
-  margin-top: -10px;
-  text-align: center;
-  display: inline-block;
-  left: 525px;
-}
-.navbar-wagon-search-btn {
-  line-height: 40px;
-  color: #E6E6E6;
-  border: none;
-  background: transparent;
-  position: absolute;
-  right: 14px;
-  top: -2px;
-  z-index: 1;
+.header-title {
+  color: white;
+}
+
+.header-subtitle {
+  color: rgba(255,255,255, 0.6);
+}
+
+.button-yellow {
+      padding: 11px 18px;
+    border: none;
+    border-radius: 3px;
+    background-color: #f5d064;
 }
+/* Navbar search form */
```

Comment:
I commented this out just in case you need this for something else, I will delete it later if not used